src/Css/Specificity/Specificity.php

```diff
@@ -80,8 +92,8 @@ public function getValues()
      */
     public static function fromSelector($selector)
     {
-        $pattern_a = "  \#";
-        $pattern_b = "  (\.[\w]+)                     # classes
+        $idSelectorsPattern = "  \#";
```

Comment:
I don't get what you mean? Can you provide me with an example where you think this will break.